README.md

```diff
@@ -1,13 +1,14 @@
 # SolidusAffirm
 
-Introduction goes here.
+This extension provides the [Affirm](https://www.affirm.com/) payment option
+for your Solidus storefront by implementing the [Affirm Direct API](https://docs.affirm.com/Integrate_Affirm/Direct_API)
 
 ## Installation
 
 Add solidus_affirm to your Gemfile:
 
 ```ruby
-gem 'solidus_affirm'
+gem 'solidus_affirm', github: 'StemboltHQ/solidus_affirm'
```

Comment:
I assume we do, although it's not published yet so the README instructions would not work. We might have to update the README once we do have a version on rubygems.